modules/indexer/code/elastic_search.go

```diff
@@ -251,6 +252,32 @@ func (b *ElasticSearchIndexer) Delete(repoID int64) error {
 	return err
 }
 
+func indexPos(content, start, end string) (int, int) {
```

Comment:
Could you add a comment to explain what it is trying to do. It's really not clear what it's supposed to be doing so I can't really assess if it's doing it correctly.